test/benchmarks/Benchmarks.Trace/HttpClientBenchmark.cs

```diff
@@ -121,17 +70,18 @@ public async Task<string> SendAsync()
                 MdToken,
                 (long)GuidPtr);
 
-            await task;
-
+            task.GetAwaiter().GetResult();
```

Comment:
just to understand: Why replacing `await task` by `task.GetAwaiter().GetResult()` ?
Is it because it not well-supported in benchmark .NET ?

Reply:
In this case is because `CallTargetSendAsync` cannot use async/await because I'm using `unsafe` in the method. So because both `CallTargetSendAsync` and `SendAsync` will be compared against each other, I want to replicate the same behavior.

Reply:
another way to solve it is using a different method for `unsafe` and calling from the benchmark method, but I didn't want to add the extra method call 😅

Reply:
Sure no pb, it was out of curiosity.
Thanks